### DISCUSSION TITLE
the InputTextMultiline function always return false

### DISCUSSION THREAD

**Discussion by SuraRengoku** [ocornut/imgui#7050] [New Users Build/Link/Run issues ONLY!]
version:1.89.6
MacOS 12.7.1
platform:xcode

I try to use the InputTextMultiline function to read the current content in buffer, there are some initial values in buffer. using a if-clause to update a u32string text when the buffer is changed, but after I changed the content in buffer, the function didn't not return a true.

the last parameter of this function is set as ImGuiInputTextFlags_EnterReturnsTure, so after I change the buffer content I click the enter Key on my keyboard, but it only worked as a "return"
<img width="542" alt="截屏2023-11-22 19 32 24" src="https://github.com/ocornut/imgui/assets/116808822/9aca177f-75f8-40bf-8dd5-c8f83b40e831">

<img width="542" alt="截屏2023-11-22 19 32 59" src="https://github.com/ocornut/imgui/assets/116808822/4423c155-736f-4786-86df-c65289ee056d">

I want to know how to make this function return a true when the buffer content is changed.

**Comment by sakiodre** [comment]
By default, `InputText` already returns true on edit, by setting the flag `ImGuiInputTextFlags_EnterReturnsTrue`, you make it return true only if the enter button is pressed, so removing that should work. You might also want to look at using the callback with `ImGuiInputTextFlags_CallbackEdit`.

**Reply by ocornut** [reply]
You may also use `IsItemEdited()` or `IsItemDeactivatedAfterEdit()` in some cases.
See Widgets->Querying Item Status->, change ItemType to InputText if you want to visualize all those values.
